src/nextcloud/talk/recording/Participant.py

```diff
@@ -246,23 +249,29 @@ def startChrome(self, width, height, env):
 
         service = ChromeService(
             env=env,
+            executable_path=driverPath,
         )
 
+        if binaryLocation is not None:
+            options.binary_location = binaryLocation
```

Comment:
Although it would not be strictly needed, for consistency with Firefox (see below) the option should be set if the string is neither `None` nor empty:
```
        if browserPath:
            options.binary_location = browserPath
```